redis-persistence/src/main/java/com/netflix/conductor/jedis/DynomiteJedisProvider.java

```diff
@@ -38,6 +47,13 @@ public DynomiteJedisProvider(
         this.tokenMapSupplier = tokenMapSupplier;
     }
 
+    private SSLContext createAndInitSSLContext() throws NoSuchAlgorithmException, KeyManagementException {
```

Comment:
This should be part of get() method of class that implements `Provider<SSLSocketFactory>`